subsys/net/ip/connection.c

```diff
@@ -576,6 +579,13 @@ enum net_verdict net_conn_input(struct net_pkt *pkt,
 	uint8_t pkt_family = net_pkt_family(pkt);
 	uint16_t src_port = 0U, dst_port = 0U;
 
+	#if defined(CONFIG_NET_FILTER)
```

Comment:
Preprocessor directives shouldn't be indented.

Reply:
Or, alternatively to ifdef-slicing the code, you could provide a dummy (empty) implementations for those functions in the header file. It's a common practice we use in net subsystem to keep the code a bit cleaner.

Reply:
Done